src/borg/crypto/low_level.pyx

```diff
@@ -113,8 +113,11 @@ cdef extern from "openssl/hmac.h":
                     const unsigned char *data, int data_len,
                     unsigned char *md, unsigned int *md_len) nogil
 
+LIBRESSL_VERSION_NUMBER = None
+
 cdef extern from "_crypto_helpers.h":
     long OPENSSL_VERSION_NUMBER
+    long LIBRESSL_VERSION_NUMBER
```

Comment:
what happens if that is not defined?

Reply:
I just learned that is causes the build to fail....